appserver/payara-appserver-modules/zendesk-support/pom.xml

```diff
@@ -22,5 +22,11 @@
             <version>${project.version}</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.main.common</groupId>
+            <artifactId>amx-core</artifactId>
+            <version>4.1.1.172-SNAPSHOT</version>
```

Comment:
this should be ${project.version}
Also why is it required, I couldn't see anything in the service that needs it?